plenum/server/node.py

```diff
@@ -2963,6 +2963,10 @@ def sum_for_values(obj):
         self.metrics.add_event(MetricsName.REPLICA_AQ_STASH_MASTER, len(self.master_replica.aqStash))
         self.metrics.add_event(MetricsName.REPLICA_REPEATING_ACTIONS_MASTER, len(self.master_replica.repeatingActions))
         self.metrics.add_event(MetricsName.REPLICA_SCHEDULED_MASTER, len(self.master_replica.scheduled))
+        self.metrics.add_event(MetricsName.REPLICA_STASHED_MASTER,
+                               self.master_replica.stasher.num_stashed_catchup +
```

Comment:
Maybe we should have separate metrics for each stashed queue?

Reply:
Yes, we should do that